.github/workflows/ci.yml

```diff
@@ -216,6 +216,7 @@ jobs:
             !:trino-hive,!:trino-orc,!:trino-parquet,
```

Comment:
Let's squash all commits. I suggest to use below for commit message:
```
Introduce connector for Apache Pulsar
```

Reply:
hmmmm, I thought all commits will be squashed to one commit when we merge the pr? is there anything I need to do here?

Reply:
you need to rebase them into one commit yourself and then force push into the branch